### PR TITLE
the option between classic and new hover tooltip

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [New] Classic tooltip can now be re-enabled via the options menu. (Though it's enabled by default from now on for the 13th of november playtest)
CL: [Improvement] The default tooltip is now more bound to the top of the screen.
CL: [Improvement] changed the font of the default tooltip.
CL: [Improvement] The classic tooltip's color is now bolder for people to be able to notice it more if they wish to enable the classic tooltip.

### CODE REVIEW

**Comment by Bod9001** [comment]
can you have the option to have the classic tooltip and the new one at the same time?

**Comment by MaxIsJoe** [comment]
> can you have the option to have the classic tooltip and the new one at the same time?

no. Though I have to question, why would you need both at the same time?

**Comment by Fr-Dae** [comment]
> 

why not ?
Redundancy is good. 
especially since we still don't know what new functionality is coming with the tooltips on top.

**Comment by Bod9001** [comment]
it seems fine to have both

**Comment by MaxIsJoe** [comment]
well this PR does not intend to add it, so maybe later with the rest of the customization options.

File: UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs [hunk 1]
```diff
@@ -74,6 +77,11 @@ void IInitialise.Initialise()
 		        PlayerPrefs.SetInt(PlayerPrefKeys.MentionSoundIndex, 0);
 	        }
 
+	        if (PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip) == false)
+	        {
+		        ClassicHoverTooltipToggle(false);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs]: couldn't this just be
```cs
ClassicHoverTooltipToggle(PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip))
```
No need for the if

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs]: I'm enabling this by default for now for next playtest

File: UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs [hunk 2]
```diff
@@ -79,7 +79,7 @@ void IInitialise.Initialise()
 
 	        if (PlayerPrefs.HasKey(PlayerPrefKeys.EnableClassicHoverTooltip) == false)
 	        {
-		        ClassicHoverTooltipToggle(false);
+		        ClassicHoverTooltipToggle(true);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs:82]: this is way too odd. What if I see the option in my settings and enable it? If you want to force it enabled, just get rid of the if or something like that.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs:82]: ah, good point. we aren't updating `IsOn` directly so it wouldn't be updated on the user's end in the options menu

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Core/OptionsMenu/ThemeOptions/ThemeManager.cs:82]: it's now enabled by default in the options menu as well

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**